com.yubico.YubiOath.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
```

Comment:
I would run `appstream-util upgrade` or just replace this with `​<component type="desktop-application">`